src/objects/npe03__Recurring_Donation__c.object

```diff
@@ -336,6 +336,18 @@
         <type>Number</type>
         <unique>false</unique>
     </fields>
+    <fields>
+        <fullName>LastElevateEventPlayed__c</fullName>
```

Comment:
@callen-sfdo , do you please mind double checking the new field on RD that will contain an Elevate Event Id? Thanks.

Reply:
@RadSFDeveloper - I assume we don't need to tell admins to put this field on their page layouts right?

Reply:
That is correct @sam-knox , the field is needed for the RD integration framework for an org connected to Elevate. It should not be added to any page layout.

Reply:
@RadSFDeveloper - As part of documentation, do we need to tell admins to add Read and Edit access to this field for their Elevate integration user, or is this going to be added to the Elevate permission sets?

Reply:
The field is relevant for the integration user, so for the regular user, the check will be in trigger that will not check field permission, so there is no need for adding permissions. When the RD MetaDeploy is run, the field will be added to the system admin and standard user profile. So, for existing Elevate customers, we can specify that they add read/edit field permission to the field.